README.md

```diff
@@ -344,24 +373,106 @@ services:
         ports:
           - "80:80"
         environment:
-          - CONNECTION_STRING=DockerMsSqlConnectionString
+          - CONNECTION_STRING=DockerComposeMSSQLConnectionString
             
     db:
         image: "mcr.microsoft.com/mssql/server"
         environment:
             SA_PASSWORD: "Qwerty1_"
             ACCEPT_EULA: "Y"
-        ports:
-          - "1433:1433"
+        expose:
+          - "1433"
 ```
 
-The application from the first container can use the following connection string to access the database:
+The application is accessible on the 80 port. However, we use the "expose" configuration for the SQL Server container which makes it reachable only inside the Docker network, and the database cannot be accessed outside.
+
+The application can use the following connection string to access the database:
 
 ```
 Pooling=false;Data Source=db;Initial Catalog=XAFContainerExample;User Id=SA;Password=<your_strong_password>
 ```
 
-Refer the [Compose specification](https://docs.docker.com/compose/compose-file/) webpage for better understanding of the Compose file format.
+If you want to run container with https support, update the `docker-compose.yml` file as follows (do not forget specify your certificate password instead of "certificate_password" example placeholder):
```

Comment:
```suggestion
If you want to run a container with HTTPS support, update the `docker-compose.yml` file as follows (remember to specify your certificate password instead of the placeholder "certificate_password"):
```